zcash_client_backend/src/scanning.rs

```diff
@@ -468,7 +471,7 @@ pub(crate) fn scan_block_with_runner<
             let spend = nullifiers
                 .iter()
                 .map(|&(account, nf)| CtOption::new(account, nf.ct_eq(&spend_nf)))
-                .fold(CtOption::new(AccountId::ZERO, 0.into()), |first, next| {
+                .fold(CtOption::new(A::default(), 0.into()), |first, next| {
```

Comment:
@str4d I continue to be highly suspicious of this constant-time-ness, and this requirement for the `Default` capability further reinforces my skepticism.